src/helpers/Environment.ts

```diff
@@ -10,9 +10,19 @@ export class EnvHelper {
    */
   static env = process.env;
   static alchemyTestnetURI = `https://eth-rinkeby.alchemyapi.io/v2/${EnvHelper.env.REACT_APP_TESTNET_ALCHEMY}`;
-
   static whitespaceRegex = /\s+/;
 
+  /**
+   * Returns env contingent segment api key
+   * @returns segment
+   */
+  static getSegmentKey() {
+    if (EnvHelper.env.NODE_ENV !== "development" && EnvHelper.env.REACT_APP_PROD_SEGMENT_KEY) {
+      return EnvHelper.env.REACT_APP_PROD_SEGMENT_KEY;
+    } else if (EnvHelper.env.REACT_APP_TESTNET_SEGMENT_KEY) {
+      return EnvHelper.env.REACT_APP_TESTNET_SEGMENT_KEY;
```

Comment:
elsewhere in my review, too, you can just use one `REACT_APP_SEGMENT_KEY` & then you don't need all this IF statement logic.